packages/pluggableWidgets/document-viewer-web/README.md

```diff
@@ -0,0 +1,5 @@
+# DocumentViewer
+
+---
+
+<!-- TODO: -->
```

Comment:
i think you can copy the content from other widgets and change path to documentviewer, cuz most probably we will add documentation for this widget sooner or later 😃

Reply:
Fixed